### ISSUE TITLE
Training for RiFe_HDv2

### ISSUE THREAD

**Issue by niklasAust** [hzwer/arXiv2020-RIFE#126]
Hello,
could you upload a sample train.py for the RIFE_HDv2 -model?

I used the current train.py to train with normal RIFE.py but when I import RIFE_HDv2 instead of import RIFE the training does not converge. With the 'normal' RIFE-model it does converge.

I used 256x448 Images and 256x448 optical Flows in both cases, with RIFE it works, with RIFE_HDv2 it does not work.

Should I use a different resolution with RIFE_HDv2?

**Comment by hzwer** [comment]
Hello,
I'm sorry for that. I fix the super parameters now. I think it will be OK.

**Comment by niklasAust** [comment]
Thank you very much,

Currently I trained RIFE for only **10** epochs with RAFT optical flows and achieved already **34.2** dB PSNR on Vimeo-90k Test Set but with more training probably much more. So results so far are promising.

I will train with LiteFlowNet in the Future and with RAFT with more epochs in the future.

**Comment by rsjjdesj** [comment]
> Thank you very much,
> 
> Currently I trained RIFE for only **10** epochs with RAFT optical flows and achieved already **34.2** dB PSNR on Vimeo-90k Test Set but with more training probably much more. So results so far are promising.
> 
> I will train with LiteFlowNet and with RAFT with more epochs in the future.
> 
> Will test the new train.py in the next week.

niklasAust:

Were you able to train RIFE_HDv2 with your dataset. And did you obtain similar or better quality than original RIFE ?

**Comment by niklasAust** [comment]
Hey,
today i trained RIFE_HDv2 for 30 epochs on Vimeo dataset and got nearly 34dB PSNR.
I had to change some superparameters, so learning was actually slower than for original RIFE, but nevertheless HDv2 seemed to converge at around 34dB PSNR.

See this issue for further information, why this could be the case: #86 

Will try RIFE-Large in the future

**Comment by hzwer** [comment]
In our train.py, we will decrease learning rate to 0 at the end of training. So the "converge" might be caused by the learning rate became too small? @niklasAust
